timemachine/fe/rbfe.py

```diff
@@ -212,7 +212,7 @@ def setup_initial_states(
     temperature: float,
     lambda_schedule: Union[NDArray, Sequence[float]],
     seed: int,
-    min_cutoff: Optional[float] = 0.7,
+    min_cutoff: Optional[float] = None,
```

Comment:
nit: slight preference for removing the default value here and explicitly passing `None` for solvent or `0.7` for complex.

Reply:
Are we sure that removing this check in the solvent leg is what we want? I'm a bit unsure; can imagine that we'd want to know if the solvent leg is sampling a different conformational mode than the complex leg

Reply:
> can imagine that we'd want to know if the solvent leg is sampling a different conformational mode than the complex leg

I think we can want to know that the solvent leg is sampling a different mode, without failing the solvent leg for it. We are currently enforcing that the solvent leg has to have the same (or at least similar) conformational mode as complex, which I don't think is generalizable.

Reply:
Going to leave the parameter as a keyword argument for now, but probably a good deboggling task.